_saas-integrations/listrak/listrak-latest.md

```diff
@@ -0,0 +1,90 @@
+---
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Listrak
+permalink: /integrations/saas/listrak
+tags: [saas_integrations]
+keywords: listrak, integration, schema, etl listrak, listrak etl, listrak schema
+layout: singer
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "listrak"
+display_name: "Listrak"
+singer: true 
+repo-url: https://github.com/singer-io/tap-listrak
+
+# this-version: ""
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+status: "Released"
+certified: false
+
+historical: "1 year"
+frequency: "30 minutes"
+tier: "Free"
+status-url: 
+icon: /images/integrations/icons/listrak.svg
+whitelist:
+  tables: true
+  columns: false
```

Comment:
I'm not aware of how we can confirm (as we don't have a test account), but I think we do support propert-selection here so this should be true I think.

Reply:
Isn't there something that's added to the tap itself to enable field selection?

Reply:
Yes. Upon closer inspection, it looks like everything is just set to "Automatic" so I think this is okay.